Frends.PowerShell.RunScript/Frends.PowerShell.RunScript/PowerShell.cs

```diff
@@ -51,6 +55,64 @@ public static PowerShellResult RunScript(RunScriptInput input, [Browsable(false)
         });
     }
 
+    private static PowerShellResult ExecuteProcess(string scriptPath, PowerShellParameter[] parameters)
+    {
+        List<dynamic> results = new();
+        List<string> errors = new();
+
+        string parameterString = string.Empty;
+        if (parameters != null)
+        {
+            foreach (var param in parameters)
+            {
+                parameterString += $"-{param.Name} \"{param.Value}\" ";
+            }
+        }
+
+        using Process process = new();
+        process.StartInfo = new()
+        {
+            FileName = "powershell.exe",
```

Comment:
I // think // that on Linux this is not going to work. I have a feeling it is something like `pwsh` there or something like that. So maybe add a check for that and use a different executable name there. Also it would be nice to add to docs that those are the executable names we expect on windows/linux.